tests/unit/src/lib/validator.test.js

```diff
@@ -71,42 +77,51 @@ const bad_length_ninos = [
   "AA",
   "A",
   "",
+  "abc def",
+  "a b c d e f",
+  "ab1cd 2",
 ];
 
 describe("should fail all the bad ninos", () => {
   test.each(bad_prefixes_ninos)(
-    "given bad prefix nino of %p, returns null string",
+    "given bad prefix nino of %p, returns false",
     (ninoArg) => {
-      const result = isValidNino(ninoArg);
-      expect(result).toBeNull();
+      testNinoValidation(ninoArg);
```

Comment:
I think this makes the test harder to understand, it would have been easiest if both validators were here side by side and you can see easily where one is false vs true for the other. Generally in testing a little bit of repeatition is a good thing